protocol-definitions/Cache.yaml

```diff
@@ -724,7 +738,7 @@ methods:
     name: removeInvalidationListener
     since: 2.0
     doc: |
-      TODO DOC
+      Removes the specified invalidation listener. Returns silently if there is no such listener added before.
```

Comment:
What does "Returns silently" mean? You may say "no change is done in the cluster and returns ..." Does it return true in that case or false?